client/lib/faux-plugins.js

```diff
@@ -224,7 +235,7 @@ module.exports = function(roomName) {
         })
         .filter(Boolean)
 
-      var playRe = /!play [^?]*\?v=([-\w]+)/
+      var playRe = /!play [^?]*\?v=([-\w]+)(&t=([0-9hms]+))?( .*)?/
```

Comment:
Well, I wanted to have a space after the time attribute, but i think it will work fine without it, i'll remove that last group, and see if it works as well.